### PR TITLE
Add Camera.get_position/size to Squirrel API.

### PR DESCRIPTION
**Pull request by tylerandari13**
This adds 4 functions to the Squirrel API.

The ability to get the Camera position. (Top left corner.)
`Camera.get_x()`
`Camera.get_y()`

The ability to get the size of the window. (In pixels)
`Camera.get_screen_width()`
`Camera.get_screen_height()`

### CODE REVIEW

**Review by Vankata453 — COMMENTED** [review]
Comments about potential `const` reference usage in getters.

File: src/object/camera.cpp [hunk 1]
```diff
@@ -866,6 +866,19 @@ Camera::get_center() const
                               static_cast<float>(m_screen_size.height) / 2.0f);
 }
 
+Vector
```
Comment (Vankata453) [inline src/object/camera.cpp:869]: Could use a `const` reference (`Vector` -> `const Vector&`).

File: src/object/camera.cpp [hunk 2]
```diff
@@ -866,6 +866,19 @@ Camera::get_center() const
...
+Camera::get_position() const
+{
+  return m_translation;
+}
+
+Size
```
Comment (Vankata453) [inline src/object/camera.cpp:875]: Could also use a `const` reference (`Size` -> `const Size&`).

File: src/object/camera.hpp [hunk 3]
```diff
@@ -100,6 +100,13 @@ class Camera final : public GameObject,
       camera */
   Vector get_center() const;
 
+  /** get the coordinates of the point directly in the top left of this
+      camera */
+  Vector get_position() const;
```
Comment (Vankata453) [inline src/object/camera.hpp:105]: Could use a `const` reference (`Vector` -> `const Vector&`).

**Review by Vankata453 — COMMENTED** [review]
Comments about potential `const` reference usage in getters.

**Review by Rusty-Box — APPROVED** [review]

**Pull request merged**